docs/ipc.md

```diff
@@ -0,0 +1,55 @@
+# IPC
+
+This documentation will guide you through the different utils provided in Fendermint for the deployment of Fendermint-based IPC subnets. This docs are only focused on the infrastructure deployment, for an end-to-end walk through of spawning IPC subnets refer to the [IPC quickstart](https://github.com/consensus-shipyard/ipc/blob/main/docs/quickstart-calibration.md).
+
+## Deploy subnet bootstrap
+In order not to expose directly the network address information from validators, subnets leverage the use of bootstraps (or `seeds` in CometBFT parlance), for new nodes to discover peers in the network and connect to the subnet's validators. To run a bootstrap node you can run the following command from the root of the repo:
+```bash
+cargo make --makefile infra/Makefile.toml bootstrap
+```
+You'll see that by the end of the output, this command should output the network address of your bootstrap. You can use this endpoint to include this bootstrap node as a seed in the `seeds` configuration of CometBFT.
+```console
+[cargo-make] INFO - Running Task: cometbft-wait
+[cargo-make] INFO - Running Task: cometbft-node-id
+2b23b8298dff7711819172252f9df3c84531b1d9@172.26.0.2:26650
+[cargo-make] INFO - Build Done in 13.38 seconds.
+```
+
+If at any time you need to query the endpoint of your bootstrap, you can run:
+```bash
+cargo make --makefile infra/Makefile.toml bootstrap-id
+```
+
+`cargo-make bootstrap` supports the following environment variables to customize the deployment:
+- `CMT_HOST_PORT` (optional): Specifies the listening port in the localhost for COMETBFT.
```

Comment:
```suggestion
- `CMT_HOST_PORT` (optional): Specifies the listening port in the localhost for CometBFT.
```

Reply:
Previously I thought this was the RPC port, but now it looks like you wanted the P2P port. Perhaps would be useful to map both so the user can access the API to hit the info URLs.

Reply:
Oh, I see. I thought I was exposing the P2P port here, but you are right that we may also want to expose the API. Can you remind me the default port for the API?